### PR TITLE
Update UpgraderTest

### PR DESCRIPTION
**Pull request by jonmv**
@mpolden please review.

### CODE REVIEW

**Comment by jonmv** [comment]
I removed some tests that were well covered by others.

File: controller-server/src/test/java/com/yahoo/vespa/hosted/controller/maintenance/UpgraderTest.java [hunk 1]
```diff
@@ -37,570 +31,431 @@
...
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
 
 /**
  * @author bratseth
  */
 public class UpgraderTest {
 
+    InternalDeploymentTester tester = new InternalDeploymentTester().atHourOfDay(5);
```
Comment (mpolden) [inline controller-server/src/test/java/com/yahoo/vespa/hosted/controller/maintenance/UpgraderTest.java]: Should be `private final`.

**Review by mpolden — COMMENTED** [review]
Great!

File: controller-server/src/test/java/com/yahoo/vespa/hosted/controller/maintenance/UpgraderTest.java (hunk 1 above)
Comment (jonmv) [reply controller-server/src/test/java/com/yahoo/vespa/hosted/controller/maintenance/UpgraderTest.java]: Yeah.

**Pull request merged**